#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_not_jpl_package.dpatch by  <javamaniac@localhost.localdomain>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: no jpl package

@DPATCH@

--- swi-prolog-5.6.13.orig/packages/configure.in
+++ swi-prolog-5.6.13/packages/configure.in
@@ -54,7 +54,7 @@
 
 
 if [ -z "$PKG" ]; then
-  PKG="clib cpp odbc table xpce sgml sgml/RDF semweb http chr clpqr nlp ssl jpl"
+  PKG="clib cpp odbc table xpce sgml sgml/RDF semweb http chr clpqr nlp ssl "
 fi
 
 configoptions=""
